packages/shared/src/components/profile/ProfileSettingsMenu.tsx

```diff
@@ -62,164 +65,178 @@ type MenuItems = Record<
-      },
-    },
-  },
-  help: {
-    title: 'Help center',
-    items: {
-      privacy: {
-        title: 'Privacy',
-        icon: PrivacyIcon,
-        href: `${settingsUrl}/privacy`,
-      },
-      reputation: {
-        title: 'Reputation',
-        icon: ReputationLightningIcon,
-        href: reputation,
-        external: true,
-      },
-      advertise: {
-        title: 'Advertise',
-        icon: MegaphoneIcon,
-        href: businessWebsiteUrl,
-        external: true,
-      },
-      apps: {
-        title: 'Apps',
-        icon: PhoneIcon,
-        href: appsUrl,
-        external: true,
-      },
-      docs: {
-        title: 'Docs',
-        icon: DocsIcon,
-        href: docs,
-        external: true,
-      },
-      support: {
-        title: 'Support',
-        icon: FeedbackIcon,
-        href: feedback,
-        external: true,
-      },
-    },
-  },
-  logout: {
-    title: null,
-    items: {
-      logout: {
-        title: 'Log out',
-        icon: ExitIcon,
-        onClick: () => logout(LogoutReason.ManualLogout),
-      },
-    },
-  },
-});
+const useAccountPageItems = () => {
+  const { openModal } = useLazyModal();
+
+  return useMemo(
+    () =>
+      defineMenuItems({
+        main: {
+          title: null,
+          items: {
+            profile: {
+              title: 'Profile details',
+              icon: UserIcon,
+              href: `${settingsUrl}/profile`,
+            },
+            account: {
+              title: 'Account',
+              icon: MailIcon,
+              href: `${settingsUrl}/security`,
+            },
+            appearance: {
+              title: 'Appearance',
+              icon: NewTabIcon,
+              href: `${settingsUrl}/appearance`,
+            },
+            notifications: {
+              title: 'Notifications',
+              icon: BellIcon,
+              href: `${settingsUrl}/notifications`,
+            },
+            invite: {
+              title: 'Invite Friends',
+              icon: InviteIcon,
+              href: `${settingsUrl}/invite`,
+            },
+          },
+        },
+        billing: {
+          title: 'Billing and Monetization',
+          items: {
+            subscription: {
+              title: 'Subscriptions',
+              icon: CreditCardIcon,
+              href: `${settingsUrl}/subscription`,
+            },
+            organization: {
+              title: 'Organizations',
+              icon: OrganizationIcon,
+              href: `${settingsUrl}/organization`,
+            },
+            coreWallet: {
+              title: 'Core Wallet',
+              icon: CoinIcon,
+              href: walletUrl,
+              external: true,
+            },
+            adsDashboard: {
+              title: 'Ads dashboard',
+              icon: TrendingIcon,
+              href: walletUrl,
+              onClick: () => openModal({ type: LazyModal.AdsDashboard }),
+            },
```

Comment:
This is the only new thing from this whole block, and the wrapping of `useMemo`.